packages/thumbprint-react/components/Image/index.jsx

```diff
@@ -84,7 +82,9 @@ const Image = forwardRef((props, outerRef) => {
     // Object Fit: polyfill and CSS styles
     // --------------------------------------------------------------------------------------------
 
-    const shouldObjectFit = height || containerAspectRatio;
+    const objectFitProps = {};
+
+    const shouldObjectFit = height;
```

Comment:
```suggestion
    const shouldObjectFit = !!height;
```

This was in the old code too, but I just noticed now.